opentracing-spring-web-itest/common/src/test/java/io/opentracing/contrib/spring/web/interceptor/itest/common/AbstractBaseITests.java

```diff
@@ -418,6 +417,16 @@ public void testControllerView() {
         assertLogEvents(span.logEntries(), Arrays.asList("preHandle", "afterCompletion"));
     }
 
+    @Test
+    public void testExcludePattern() throws InterruptedException {
+        {
+            getRestTemplate().getForEntity("/health", String.class);
+            Thread.sleep(505);
```

Comment:
Isn't there a better way to do that? Like, using Powermock to ensure a method is *not* called?